helm/flink-kubernetes-operator/templates/flink-operator.yaml

```diff
@@ -110,6 +110,17 @@ spec:
               value: -Dlog4j.configurationFile=/opt/flink/conf/log4j-operator.properties
             - name: JVM_ARGS
               value: {{ .Values.jvmArgs.operator }}
+            {{- if .Values.tls.create }}
+            - name: OPERATOR_KEYSTORE_PATH
+              value: /opt/flink/artifacts/operator-cert/keystore.jks
+            - name: OPERATOR_TRUSTSTORE_PATH
+              value: /opt/flink/artifacts/operator-cert/truststore.jks
+            - name: OPERATOR_KEYSTORE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  {{- toYaml .Values.tls.certPasswordSecretRef | nindent 18 }}
```

Comment:
can we call this property `.Values.tls.secretKeyRef` ? that way it's obvious what is expected to go there